README.md

```diff
@@ -122,6 +122,32 @@ You can then build using the registry cache with the command:
 kubectl build -t myimage --cache-to=type=registry,ref=registry:5000/cache --cache-from=type=registry,ref=registry:5000/cache .
 ```
 
+## Custom Certs for Registries
+
+If you happen to run a container image registry with non-standard certs (self signed, or signed by a private CA)
+you can configure buildkitd to trust those certificates.
+
+First load your certs into a ConfigMap:
+```sh
+kubectl create configmap custom-root-certs --from-file=root_ca1.pem --from-file=root_ca2.pem
+```
+
+Then create a `buildkitd.toml` file for your registries where the certs above will be mounted into `/etc/config`
```

Comment:
Looking through the code below, seems like they get mounted into "/etc/config", so this is kind of a non-starter I think?

Reply:
While it may be possible, in general, you want to mount things into a directory that's not already present.  I was also aiming to emulate the upstream doc's around examples for certs  (scroll to the bottom of https://github.com/moby/buildkit/blob/master/docs/buildkitd.toml.md )